src/pages/gen1/index.tsx

```diff
@@ -42,15 +45,18 @@ const PlatformOverview = () => {
             variation="primary"
             size="large"
             href={{
-              pathname: '/[platform]/how-amplify-works',
+              pathname: '/gen1/[platform]/how-amplify-works',
```

Comment:
should we store this in a shared const somewhere in case we need to update it across our tsx files? I think it makes sense to be hardcoded in mdx, but in tsx files it would be nice if we didn't have to do find/replace, what do you think?